remix/app/lib/toast/makeToast.ts

```diff
@@ -0,0 +1,15 @@
+import toast from 'react-hot-toast';
+
+export default function makeToast(
+  toastMode: string | null,
+  toastMessage: string | null
```

Comment:
Don't you think we could pass an object instead of having two attributes:

```suggestion
export default function showNotification({
  text: string
  type?: string,
}  : Notification)
```

> **Note**
> The attribute text should not be nullable IMO. The interface toastItems (renamed to Notification) could be define here instead.

Reply:
Yep, Passing an object would be better than two attributes. Thank you for mentioning that 😅 🙏🏼 

Improved: [f078d2a](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/f078d2aa6e5b90e3f455452ffa7e67ef3fd64881)